CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - docs: add docs around `Permission denied` issues ([#28](https://github.com/heroku/nodejs-npm-buildpack/pull/28))
 - Add dockerized unit tests ([#29](https://github.com/heroku/nodejs-npm-buildpack/pull/29))
 - Added `provides` and `requires` of `node_modules` and `node` to buildplan. ([#18](https://github.com/heroku/nodejs-npm-buildpack/pull/18))
+- Prune devdependencies ([#32](https://github.com/heroku/nodejs-npm-buildpack/pull/32))
```

Comment:
Can you move this to under `master` since it's going in the next version change? (And add an `### Added` header :) )